admin_apps/requirements.txt

```diff
@@ -5,4 +5,4 @@ sqlglot==23.17.0
 streamlit==1.36.0
 streamlit-extras==0.4.3
 streamlit-monaco==0.1.3
-snowflake-connector-python==3.11.0
\ No newline at end of file
+snowflake-connector-python[secure-local-storage]==3.11.0
```

Comment:
Need `secure-local-storage` to fully support [MFA token caching](https://docs.snowflake.com/en/user-guide/security-mfa#label-mfa-token-caching).